examples/sundials/ex16.cpp

```diff
@@ -1,20 +1,24 @@
 //                                MFEM Example 16
 //                             SUNDIALS Modification
 //
-// Compile with: make ex16
+// Compile with: make sundials_ex16
 //
-// Sample runs:  ex16
-//               ex16 -m ../../data/inline-tri.mesh
-//               ex16 -m ../../data/disc-nurbs.mesh -tf 2
-//               ex16 -s 12 -a 0.0 -k 1.0
-//               ex16 -s 8 -a 1.0 -k 0.0 -dt 1e-4 -tf 5e-2 -vs 25
-//               ex16 -s 9 -a 0.5 -k 0.5 -o 4 -dt 1e-4 -tf 2e-2 -vs 25
-//               ex16 -s 10 -dt 1.0e-4 -tf 4.0e-2 -vs 40
-//               ex16 -m ../../data/fichera-q2.mesh
-//               ex16 -m ../../data/escher.mesh
-//               ex16 -m ../../data/beam-tet.mesh -tf 10 -dt 0.1
-//               ex16 -m ../../data/amr-quad.mesh -o 4 -r 0
-//               ex16 -m ../../data/amr-hex.mesh -o 2 -r 0
+// Sample runs:  sundials_ex16
+//               sundials_ex16 -m ../../data/inline-tri.mesh
+//               sundials_ex16 -m ../../data/disc-nurbs.mesh -tf 2
```

Comment:
The switch in the sample runs `ex16` -> `sundials_ex16` seems motivated by the naming used in the CMake build system. However, the sample runs are assumed to be consistent with the GNU make build system, at least in some testing scripts, e.g. `config/sample-runs.sh` and a similar script used by the nightly tests.

Maybe we can unify the naming of the executable in both GNU make and in CMake -- currently CMake uses the name of the target, `sundials_ex16`, which has to be different from `ex16` to distinguish it from the target for `examples/ex16`. However, we should be able to set the executable name to simply `ex16` since it is in the `examples/sundials` sub-directory. What do you think @cjvogl?

Reply:
I like your idea!  I'll make the change to the `CMakeLists.txt` and revert the sample runs.

Reply:
This should be done: thanks again for the suggestion.